### DISCUSSION TITLE
Strange bug in one thermostat

### DISCUSSION THREAD

**Discussion by M-ERiMiEs** [ScratMan/HASmartThermostat#230] [General]
Hey, im using smart thermostat on five rooms and all other seems to work right except the "Olohuone" which doesnt get the preset temperature for Home-home. Itll go to 21 even if its enabled like bellow. On other rooms preset temps works right and only stange thing is that if I enable boost-mode it also enables home-mode ie in "Makuuhuone 3" thermostat config.

- platform: smart_thermostat
  name: Olohuone
  unique_id: Olohuone
  heater:
    - switch.rele_1_sonoff_4ch_relay_1
    - switch.rele_1_sonoff_4ch_relay_2
    - switch.rele_1_sonoff_4ch_relay_3
  target_sensor: sensor.olohuone_temperature
  outdoor_sensor: sensor.vallox_ulkoilma
  min_temp: 18
  max_temp: 24
  ac_mode: False
  target_temp: 22
  home_temp: 22
  boost_temp: 24
  keep_alive:
    seconds: 60
  kp: 40
  ki: 0.001
  kd: 30000
  ke: 0.6
  pwm: 00:30:00
  min_cycle_duration: 00:05:00
  force_off_state: false
  boost_pid_off: true
  
- platform: smart_thermostat
  name: Makuuhuone 3
  unique_id: Makuuhuone_3
  heater: switch.rele_1_sonoff_4ch_relay_4
  target_sensor: sensor.makuuhuone_3_temperature
  outdoor_sensor: sensor.vallox_ulkoilma
  min_temp: 18
  max_temp: 24
  ac_mode: False
  target_temp: 22
  boost_temp: 24
  keep_alive:
    seconds: 60
  kp: 40
  ki: 0.001
  kd: 30000
  ke: 0.6
  pwm: 00:30:00
  min_cycle_duration: 00:05:00
  force_off_state: false
  boost_pid_off: true

**Comment by ScratMan** [comment]
Hi, preset temperatures in YAML are only used at init, the last value is restored upon reboot and can be set using set_preset_temp service.

**Reply by M-ERiMiEs** [reply]
As I write it already I found out. Now it seems to work right.

For same post I have new feature suggestion for anti jamming for valves etc as they do not operate thru summer.
